### PR TITLE
[azure] add install workflow and ci jobs for single node on azure

### PR DESCRIPTION
**Pull request by jinyunma**
[OCPBU-460](https://issues.redhat.com/browse/OCPBU-460?atlOrigin=eyJpIjoiYjM0MTA4MzUyYTYxNDVkY2IwMzVjOGQ3ZWQ3NzMwM2QiLCJwIjoianN3LWdpdGxhYlNNLWludCJ9) Support OpenShift 4.x SNO on Azure

### CODE REVIEW

**Comment by jinyunma** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installer-rehearse-4.14-installer-rehearse-azure

**Comment by jinyunma** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installer-rehearse-4.14-installer-rehearse-azure

**Comment by jinyunma** [comment]
/unhold

@jianlinliu could you help to review again? Add relevant ci jobs.

**Comment by jianlinliu** [comment]
cc @liangxia to review.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml [hunk 1]
```diff
@@ -1644,6 +1644,30 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-azure-ipi-sdn
+- as: azure-ipi-sno-p2-f28
+  cron: 19 10 19 * *
+  steps:
+    cluster_profile: azure-qe
+    env:
+      BASE_DOMAIN: qe.azure.devcluster.openshift.com
+      E2E_RUN_TAGS: '@amd64 and @azure-ipi and @network-ovnkubernetes and not @fips'
```
Comment (liangxia) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml:1653]: For SNO, we need tag `@singlenode`

Comment (jinyunma) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml]: Fixed.

**Pull request merged**